packaging/machine-learning-api.spec

```diff
@@ -139,6 +194,12 @@ meson --buildtype=plain --prefix=%{_prefix} --sysconfdir=%{_sysconfdir} --libdir
 
 ninja -C build %{?_smp_mflags}
 
+# Run test
+%if 0%{?unit_test}
+bash %{test_script} ./tests/capi/unittest_capi_inference
+bash %{test_script} ./tests/capi/unittest_capi_inference_nnfw_runtime
+%endif
```

Comment:
Don't run `unittest_capi_inference_latency`?

Reply:
unittest_capi_inference_latency is benchmark tool, to measure invoke time.
It contains single-shot invoke function with imgclf model and this is already in unittest_capi_inference.